static/js/click_handlers.js

```diff
@@ -653,7 +653,17 @@ export function initialize() {
             e.stopPropagation();
         }
         // Still hide the popovers, however
-        popovers.hide_all();
+        if ($(e.target).is("textarea")) {
+            // This stops formatting buttons popover from hiding when
+            // user selects a range of text in compose box in Firefox.
+            // In Firefox, this event is trigger after popover is
+            // triggered to show while in chrome, it triggers before the popover shows.
+            popovers.hide_all({
+                not_hide_tippy_instances: true,
+            });
+        } else {
+            popovers.hide_all();
+        }
```

Comment:
Could we just pass `not_hide_tippy_instances` unconditionally in this code path?  

I also wonder if we want a different name/semantics for controlling that hiding logic, e.g. a more generic `except_popover: {popover_name}`?  Not sure.

Reply:
Yeah, done the expect_popover thing.

Reply:
I think this might not work; in that `popovers.hide_all` seems to not pass through the parameter properly.  Probably needs a preparatory refactoring to correct that.